tuned/plugins/plugin_scheduler.py

```diff
@@ -1098,10 +1098,13 @@ def _thread_code(self, instance):
 						event = instance._evlist.read_on_cpu(cpu)
 						if event:
 							read_events = True
-							if event.type == perf.RECORD_COMM or \
-								(self._perf_process_fork_value and event.type == perf.RECORD_FORK):
+							if isinstance(event, perf.comm_event) or (
```

Comment:
There could be explicit sanity check that `event.type == perf.RECORD_COMM`, but taking into account the perf interface is stable for years it's unlikely that some future perf update would break this "optimization".